### PR TITLE
add Whive (WHIVE)

### PR DESCRIPTION
**Pull request by whiveio**

### CODE REVIEW

File: assets/config/0.5.4-coins.json [hunk 1]
```diff
@@ -9345,6 +9345,40 @@
...
+      {
+        "url": "electrumx2.cointest.com:50001",
+        "protocol": "TCP"
+      }
+    ],
+    "explorer_url": [
+      "https://whiveexplorer.cointest.com/"
+    ],
+    "type": "UTXO",
+    "active": true,
```
Comment (smk762) [inline assets/config/0.5.4-coins.json:9378]: pls set to false

File: assets/config/0.5.4-coins.json [hunk 2]
```diff
@@ -9345,6 +9345,40 @@
...
+    "is_segwit_on": true,
```
Comment (smk762) [inline assets/config/0.5.4-coins.json:9379]: pls set to false

Comment (nicholas-mwaura) [reply assets/config/0.5.4-coins.json:9379]: i set both "active' and "is_segwit_on" to false ?

**Review by cipig — APPROVED** [review]

**Review by smk762 — APPROVED** [review]

**Comment by whiveio** [comment]
Merge was not successful for [ubuntu-qt-5-15-2](https://github.com/KomodoPlatform/atomicDEX-Desktop/pull/1611#logs) and [osx-qt-5-15-2](https://github.com/KomodoPlatform/atomicDEX-Desktop/pull/1611#logs)

**Comment by tonymorony** [comment]
on OSX it failing because codesigning part works only for PRs opened by maintainers, most probably Ubuntu fails by the similar reason since it's building fine locally

**Comment by cipig** [comment]
the SSL certificates are not accepted by mm2
`coins:rpc_clients:2046] ERROR "electrumx1.cointest.com:50002" error Custom { kind: InvalidData, error: WebPKIError(BadDER) }`
it's likely because they are self-signed. Those don't work, only real ones, eg from letsencrypt.

**Comment by nicholas-mwaura** [comment]
put in real SSL certificates(letsencrypt)  for both electrumx servers :-  electrumx1.cointest.com and electrumx2.cointest.com:

**Comment by cipig** [comment]
thanks a lot, works fine now

**Comment by nicholas-mwaura** [comment]
hi,
When is the next release of the desktop app?

**Pull request merged**